packages/noir-ethereum-api/test/e2e.test.ts

```diff
@@ -20,41 +20,43 @@ const defaultTestCircuitInputParams: MainInputs = {
     '0xda', '0xc4', '0x8e', '0x2b', '0x4']
 };
 
+function alterCall(callName: string, path: string[]) {
```

Comment:
Let's enable a linter rule that each function should have a return type specified
https://app.asana.com/0/1206416222603091/1206417670340098